src/main/java/org/jenkinsci/plugins/mesos/JenkinsScheduler.java

```diff
@@ -483,6 +484,19 @@ private boolean matches(Offer offer, Request request) {
     }
   }
 
+  private boolean buildsInQueue(hudson.model.Queue queue) {
+   hudson.model.Queue.Item[] items =  queue.getItems();
```

Comment:
getApproximateItemsQuickly()
Deprecated. Use getItems() directly. As of 1.607 the approximation is no longer needed.
Like getItems(), but returns an approximation that might not be completely up-to-date.